pytorch_toolkit/ote/ote/evaluation/common.py

```diff
@@ -7,9 +7,8 @@
 
 import yaml
 from mmcv import Config
-from oteod import MMDETECTION_TOOLS
-from oteod.misc import get_complexity_and_size
-from ote.misc import get_file_size_and_sha256
+from ote import MMDETECTION_TOOLS
+from ote.misc import get_file_size_and_sha256, get_complexity_and_size
 
 
```

Comment:
I think it's better to move all task-specific metric definitions to the related tasks (for example `collect_ap`, `coco_ap_eval` into `oteod` or something instead of it).

Reply:
Or create separate `metrics.py` file but define the list of metrics for particular model individually.

Reply:
Do you mean metrics different from COCO? And what should be in `metrics.py` file?

Reply:
I mean that each owner of candidate `mm-xxx` repo will add task-specific metrics (mAP for detection, accuracy for classification and so on) into this file. Obviously `metrics.py` should contain some registry of metrics to find them by name.